sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -422,7 +422,13 @@ export class BatchingReceiver extends MessageReceiver {
         });
         this._init(rcvrOptions)
           .then(() => {
-            this._receiver!.on(ReceiverEvents.receiverDrained, onReceiveDrain);
+            if (this._receiver == null) {
```

Comment:
In order to follow similar patterns as other places in the library...

```suggestion
            if (!this._receiver) {
```